aiohttp_sse/__init__.py

```diff
@@ -172,26 +194,59 @@ async def _ping(self):
             try:
                 await self.write(": ping{0}{0}".format(self._sep).encode("utf-8"))
             except ConnectionResetError:
-                self._ping_task.cancel()
+                if self._ping_task is not None:  # pragma: no cover (made for mypy)
```

Comment:
This `if` I made for mypy only.
Another way coverage report marks this line as redundant, cause condition `always False`

Reply:
If we can gaurantee that it is always the case, then change it to an assert, rather than skipping coverage.

Reply:
I think you removed this code.